### DISCUSSION TITLE
Access getAuth() from Firebase SDK

### DISCUSSION THREAD

**Discussion by borjalo** [gladly-team/next-firebase-auth#530] [Q&A]
I think that something is not working property. When I try to access the auth object from `firebase/auth` in my `login.tsx` it seems to not be initialized yet so that's why it's crashing. How can I wait until the `initAuth()` is executed before calling the `getAuth() ` from Firebase SDK?

This is `login.tsx`:
<img width="524" alt="image" src="https://user-images.githubusercontent.com/22989690/185468911-6535ba75-7f18-4f6d-851e-4ea1ecac903c.png">

And this is my `_app.tsx`
<img width="310" alt="image" src="https://user-images.githubusercontent.com/22989690/185468952-25cd9eee-b697-4267-8ecd-b3953996d105.png">

**Comment by kmjennison** [comment]
The init method is synchronous, so you shouldn't have to wait for it. Additional info in this discussion: https://github.com/gladly-team/next-firebase-auth/discussions/526#discussioncomment-3426320
